app.css

```diff
@@ -51,15 +54,17 @@ body {
   text-decoration: none;
   color: #172b4d;
   font-style: normal;
-  font-size: 40px;
   /* stylelint-disable-next-line csstree/validator */
   font-weight: 700, bold;
 }
 .personal-info-detail p {
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly ensure you have the appropriate background image as we have on the template in the Intro section of your portfolio.

 - [ ] As suggested by the previous reviewer, kindly place the 'Glad to see you!' text on the next line as we have on the template, kindly do that.

- Your design:
<img width="1120" alt="Screenshot 2022-04-02 at 6 45 37 AM" src="https://user-images.githubusercontent.com/70269507/161364971-75c9ebdb-d6a6-4976-9eb8-1d280a5b15e6.png">

- Figma design:
<img width="693" alt="Screenshot 2022-04-02 at 6 45 03 AM" src="https://user-images.githubusercontent.com/70269507/161364978-95491405-bbf9-42f9-81d5-948ff92cb78d.png">